test/functional/testUtil/diffHtml.js

```diff
@@ -9,7 +9,9 @@ const DiffPrinter = require('./diffPrinter');
  * @returns {bool} if diff was found
  */
 const diffHtml = (expected, actual, filePathName) => {
-  const diffParts = jsdiff.diffWords(expected, actual);
+  // ignore default template's footer
+  const timeRegex = /\[Generated by .*? on .*?\]/;
```

Comment:
Could we use an empty footer instead?
This change may cause text explicitly typed by the user that matches the regex to not be detected.

Looks good otherwise 👍

Reply:
If that line is not present, five of the tests fail.

Reply:
Yes, because `{{ timestamp }}` is present in the footer, those tests would fail.
You could make the footer empty ( but keep the `<footer>` tag ) to fix it

Reply:
Just noticed `src/template/minimal` has the same `footer/header/navgation.md` files too, do update those as well

Reply:
@ang-zeyu I only needed to take out one line to make all the tests pass (i.e. the footer remains non-empty). Please re-review.

And by the way, what files am I supposed to change to make the footer empty?

Reply:
> And by the way, what files am I supposed to change to make the footer empty?

That would be the same `footer.md` file you already edited, but removing the `{{ timestamp }}` is fine as well 👍 

Do make the changes for the minimal template's files too ( `src/template/minimal` )